frontend/src/Pages/Movie.js

```diff
@@ -76,16 +77,16 @@ function Movie(props) {
     return newDate;
   }
 
-  const [favButtonText, setFavButtonText] = useState("Add to Favorites");
-  const [favButtonDisabled, changeDisabled] = useState(false);
+  const [favButtonText, setFavButtonText] = useState("Add To Favorites");
+  const [favButtonVariant, setFavButtonVariant] = useState("danger");
```

Comment:
Is the variant ever not using danger? I don't see where this is used

Reply:
I was trying to make the button change colors when it switches from "Add To Favorites" to "Favorited". That is the purpose of coding it in this fashion.